src/requirements/data/minors/ineq.ts

```diff
@@ -0,0 +1,235 @@
+import { CollegeOrMajorRequirement } from '../../types';
```

Comment:
I think this file got included from your other PR as a mistake. Could you please remove it from this branch?